frontend/public/components/workload-table.tsx

```diff
@@ -40,9 +42,13 @@ export const WorkloadTableRow: React.FC<WorkloadTableRowProps> = ({obj, index, k
         <LabelList kind={kind} labels={obj.metadata.labels} />
       </TableData>
       <TableData className={tableColumnClasses[3]}>
-        <Link to={`${resourcePath(kind, obj.metadata.name, obj.metadata.namespace)}/pods`} title="pods">
-          {obj.status.replicas || 0} of {obj.spec.replicas} pods
-        </Link>
+        { _.get(obj, ['metadata', 'annotations', 'openshift.io/deployment.cancelled']) ?
```

Comment:
I'd add a separate column and the details page property for replication controllers that have `openshift.io/deployment.phase` that just shows the phase as-is.

Reply:
So the `openshift.io/deployment.phase` of a cancelled Replication Controller is Failed. I don't know if the phase has enough meaningful information to warrant its own column. I think the cancelled flag set to true is the only way we know an RC was manually cancelled.

Reply:
We are adding phase not just for the cancelled case, but for all cases. It could be Running, Completed, etc. Cancelled is rare, so we shouldn't optimize the list view for that case.